blocks/article-tag-block/features/tag/default.test.jsx

```diff
@@ -1,144 +1,148 @@
-describe("This test is disabled", () => {
-	it("should succeed", () => {
-		expect(true);
+import { render, screen } from "@testing-library/react"
+import { useFusionContext } from "fusion:context";
+import React from "react";
+import ArticleTags from "./default";
+import '@testing-library/jest-dom'
+
+jest.mock("@wpmedia/arc-themes-components", () => ({
+	...jest.requireActual("@wpmedia/arc-themes-components"),
+	isServerSide: jest.fn().mockReturnValue(true),
+	LazyLoad: ({ children }) => children,
+}));
+
+describe("the article tag block", () => {
+	describe("when the global content has an array of tags in its taxonomy", () => {
+		const mockReturnData = {
+			arcSite: "the-sun",
+			globalContent: {
+				taxonomy: {
+					tags: [
+						{
+							description: "dogs",
+							slug: "dogs slug",
+							text: "dogs text",
+						},
+						{
+							description: "cats",
+							slug: "cats slug",
+							text: "cats text",
+						},
+					],
+				},
+			},
+		};
+
+		afterEach(() => {
+			jest.resetModules();
+		});
+
+		beforeEach(() => {
+			useFusionContext.mockReturnValue(mockReturnData);
+		});
+
+		test("should return null if lazyLoad on the server and not in the admin", async () => {
+			const config = {
+				lazyLoad: true,
+			};
+			render(<ArticleTags customFields={config} />);
+			const wrapper = await screen.findByRole("generic");
+			expect(wrapper.innerHTML).toBeFalsy();
```

Comment:
`expect(wrapper).toBeEmptyDOMElement()` is a better way to check for empty elements. It should be available since this test file imports `@testing-library/jest-dom`

Reply:
@vgalatro Fixed it